addons/auth_signup_turnstile/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+{
+    'name': 'Signup Turnstile',
+    'description': """
+Implements Turnstile for the signup and reset password page.
+============================================================
+    """,
+    'version': '1.0',
+    'category': 'Hidden/Tools',
+    'auto_install': True,
+    'installable': True,
+    'depends': [
+        'website_cf_turnstile',
```

Comment:
My previous point regarding this dependence still stand

> I also see that you add website_cf_turnstile as a dependence of auth_sigup_turnstile, meaning if you want turnstile for the signup/reset password, you force people to install the website. In addition to be weird, it means that you will force an upsell to customers who want the turnstile protection and who do not have/pay for the website module. As the pricing is done by module.

In shorter words, you shouldn't make customers pay for the website module to be able to be protected by turnstile.

The signup/reset password forms should be able to be protected by turnstile without having the website module installed, in my opinion.

If it's not easily feasible, I think we discussed it's not normal to have free signup activated without a website, and it's kinda true. But even if you disable the free signup, you can still spam mails with the reset password form no ? So you might still want to be able to have turnstile without website and without free signup, no ?